packages/astro/test/fixtures/astro-scripts/src/pages/inline.astro

```diff
@@ -12,5 +12,9 @@ import Inline from '../components/Inline.astro';
   <Inline />
   <Inline />
   <Inline />
+	<script>
+		// This must be relative for the test
+		import '../scripts/hoist_external_imported_inline.js';
```

Comment:
Can you update the test suite to verify the script was hoisted and bundled?

Reply:
Yeah, for the sake of this bug it would just fail to build on Windows. But I can do a verification as well.